indexer/editable_map_object.hpp

```diff
@@ -55,8 +55,8 @@ class EditableMapObject : public MapObject
 
   StringUtf8Multilang const & GetName() const;
   vector<LocalizedName> GetLocalizedNames() const;
-  string const & GetStreet() const;
-  vector<string> const & GetNearbyStreets() const;
+  pair<string, string> const & GetStreet() const;
```

Comment:
using LocalizedStreet будет лучше

> On Apr 8, 2016, at 15:32, Vlad Mihaylenko <notifications@github.com> wrote:
> 
> In indexer/editable_map_object.hpp <https://github.com/mapsme/omim/pull/2850#discussion_r59017611>:
> 
> > @@ -55,8 +55,8 @@ class EditableMapObject : public MapObject
> >  
> >    StringUtf8Multilang const & GetName() const;
> >    vector<LocalizedName> GetLocalizedNames() const;
> > -  string const & GetStreet() const;
> > -  vector<string> const & GetNearbyStreets() const;
> > +  pair<string, string> const & GetStreet() const;
> если @deathbaba <https://github.com/deathbaba> не против, я бы тоже сделал
> using Street = pair;
> using Streets = vector;
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/2850/files/0ad17160bff832a1d3538b656f12f37df22310a9#r59017611>